<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>templex::frontend::FunctionInstantiationsPage</class>
 <widget class="QWidget" name="templex::frontend::FunctionInstantiationsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="templatesLabel">
       <property name="text">
        <string>Templates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>filter name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterEdit"/>
     </item>
     <item>
      <widget class="QListView" name="templateList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="overloadsLabel">
       <property name="text">
        <string>Overloads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="overloadsList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="instantiationsLabel">
       <property name="text">
        <string>Instantiations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
